fed/api.py

```diff
@@ -33,7 +33,11 @@
     send,
     start_recv_proxy,
     start_send_proxy,
+    SendProxy,
+    RecvProxy
 )
+from fed.config import CrossSiloCommConfig
```

Comment:
I believe the more native name is `CommonCrossSiloConfig`

Reply:
"Comm" is short for "Communication" 😅

Reply:
😅 So that we should even more not use it as it's confusing even in this conversation.

Reply:
Rename to `CrossSiloMsgConfig` and `GrpcCrossSiloMsgConfig`, any other name suggestions?

Reply:
proposing `CrossSiloMessageConfig`